### PR TITLE
Changelog for v2.88.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,63 @@ Synapse Changelog
...
+    Change the type of the ``:hash`` property from a ``0x`` prefixed ``str``
+    to a ``hash`` type.
+    Deprecate the ``:inputs`` and ``:outputs`` secondary properties.
+
+  ``geo:place``
+    Change the type of the ``:name`` secondary property to ``geo:name``.
+
+- Add support for ``$lib.len()`` to count the length of emitter or generator
+  functions.
+  (`#2606 <https://github.com/vertexproject/synapse/pull/2606>`_)
```
Comment (Cisphyx) [inline CHANGELOG.rst:50]: Wrong PR, should be 2603

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,63 @@ Synapse Changelog
...
+    Add a secondary property ``:transaction`` to denote the transaction
+    for the payment.
+
+  ``crypto:payment:input``
+    Add a secondary property ``:transaction`` to denote the transaction
+    for the payment.
+
+  ``crypto:currency:block``
+    Change the type of the ``:hash`` property from a ``0x`` prefixed ``str``
+    to a ``hash`` type.
```
Comment (invisig0th) [inline CHANGELOG.rst:38]: Did you mean a `hex` type here?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,63 @@ Synapse Changelog
...
+
+  ``crypto:currency:transaction``
+    Change the type of the ``:hash`` property from a ``0x`` prefixed ``str``
+    to a ``hash`` type.
```
Comment (invisig0th) [inline CHANGELOG.rst:42]: ( and here )

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,63 @@ Synapse Changelog
...
+Features and Enhancements
+-------------------------
+- Several updates for the ``crypto`` and ``geospace`` models.
+  (`#2594 <https://github.com/vertexproject/synapse/pull/2594>`_)
+
+  ``crypto:payment:input``
+    Add a secondary property ``:transaction`` to denote the transaction
+    for the payment.
+
+  ``crypto:payment:input``
```
Comment (whippit) [inline CHANGELOG.rst:32]: crypto:payment:output?

**Review by invisig0th — CHANGES_REQUESTED** [review]

**Review by invisig0th — APPROVED** [review]

**Pull request merged**